### PR TITLE
Update i18n.js french

### PR DESCRIPTION
**Pull request by Zac0511**
I added the french to the file

### CODE REVIEW

**Comment by Zac0511** [comment]
I just modified it because i forgot the first thing (not translated because i didn't see it the first time

File: src/i18n/i18n.js [hunk 1]
```diff
@@ -614,4 +614,146 @@ window.translations = {
         you_have_been_referred_to_puter_by_a_friend: "您已经被朋友推荐到 Puter！",
         zip: "压缩",
     },
-}
\ No newline at end of file
+    //french
+    fr: {
+        access_granted_to: "Accès accordé à",
+        add_existing_account: "Ajouter un compte existant",
+        all_fields_required: 'All fields are required.',
```
Comment (mael-belval) [inline src/i18n/i18n.js:621]:
```suggestion
        all_fields_required: 'Tous les champs sont requis.',
```

File: src/i18n/i18n.js [hunk 2]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        apply: "Appliquer",
+        ascending: 'Ascending',
+        background: "Background",
```
Comment (mael-belval) [inline src/i18n/i18n.js:624]:
```suggestion
        background: "Arrière-plan",
```

File: src/i18n/i18n.js [hunk 3]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
     },
-}
\ No newline at end of file
+    //french
+    fr: {
+        access_granted_to: "Accès accordé à",
+        add_existing_account: "Ajouter un compte existant",
+        all_fields_required: 'All fields are required.',
+        apply: "Appliquer",
+        ascending: 'Ascending',
```
Comment (mael-belval) [inline src/i18n/i18n.js:623]:
```suggestion
        ascending: 'Croissant',
```

File: src/i18n/i18n.js [hunk 4]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        background: "Background",
+        browse: "Browse",
```
Comment (mael-belval) [inline src/i18n/i18n.js:625]:
```suggestion
        browse: "Explorer",
```

File: src/i18n/i18n.js [hunk 5]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        cancel: 'Annuler',
+        center: 'Center',
```
Comment (mael-belval) [inline src/i18n/i18n.js:627]:
```suggestion
        center: 'Centrer',
```

File: src/i18n/i18n.js [hunk 6]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        downloading: "Téléchargement en cours",
+        email: "Email",
+        email_or_username: "Email ou nom d'utilisateur",
+        empty_trash: 'Vider la corbeille',
+        empty_trash_confirmation: `Êtes-vous sûr de vouloir supprimer définitivement les éléments de la corbeille ?`,
+        emptying_trash: 'Vidage de la corbeille...',
+        feedback: "Envoyer des commentaires",
+        feedback_c2a: "Veuillez utiliser le formulaire ci-dessous pour nous envoyer vos retours, commentaires et rapports de bugs.",
+        feedback_sent_confirmation: "Merci de nous avoir contactés. Si vous avez un e-mail associé à votre compte, vous recevrez une réponse de notre part dans les plus brefs délais.",
+        forgot_pass_c2a: "Mot de passe oublier?",
```
Comment (mael-belval) [inline src/i18n/i18n.js:665]:
```suggestion
        forgot_pass_c2a: "Mot de passe oublié ?",
```

File: src/i18n/i18n.js [hunk 7]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        publish: "Publier",
+        publish_as_website: 'Publier en tant que site internet',
+        recent: "Récent",
+        recover_password: "Récupérer le mot de passe",
+        refer_friends_c2a: "Obtenez 1 Go pour chaque ami qui crée et confirme un compte sur Puter. Votre ami recevra également 1 Go !",
+        refer_friends_social_media_c2a: `Obtenez 1 Go de stockage gratuit sur Puter.com !`,
+        refresh: 'Actualiser',
+        release_address_confirmation: `Êtes-vous sûr de vouloir libérer cette adresse ?`,
+        remove_from_taskbar:'Retirer de la barre des tâches',
+        rename: 'Renomer',
```
Comment (mael-belval) [inline src/i18n/i18n.js:720]:
```suggestion
        rename: 'Renommer',
```

File: src/i18n/i18n.js [hunk 8]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        repeat: 'Répéter',
+        resend_confirmation_code: "Renvoyer le code de confirmation",
+        restore: "Restorer",
+        save_account_to_get_copy_link: "Veuillez créer un compte pour continuer",
+        save_account_to_publish: 'Veuillez créer un compte pour continuer.',
+        save_session_c2a: 'Créez un compte pour sauvegarder votre session en cours et éviter de perdre votre travail.',
+        scan_qr_c2a: 'Scannez le code ci-dessous pour vous connecter à cette session depuis d'autres appareils',
```
Comment (mael-belval) [inline src/i18n/i18n.js:727]:
```suggestion
        scan_qr_c2a: "Scannez le code ci-dessous pour vous connecter à cette session depuis d'autres appareils",
```

File: src/i18n/i18n.js [hunk 9]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        no_dir_associated_with_site: 'Aucun répertoire associé à cette adresse.',
+        no_websites_published: "Vous n'avez pas encore publié de sites internet.",
+        ok: 'OK',
+        open: "Ouvrir",
+        open_in_new_tab: "Ouvrir dans un nouvel onglet",
+        open_in_new_window: "Ouvrir dans une nouvelle fenêtre",
+        open_with: "Ouvrir avec",
+        password: "Mot de passe",
+        password_changed: "Mot de passe modifié.",
+        passwords_do_not_match: ''Nouveau mot de passe' et 'Confirmer le nouveau mot de passe' ne correspondent pas.',
```
Comment (mael-belval) [inline src/i18n/i18n.js:702]:
```suggestion
        passwords_do_not_match: "'Nouveau mot de passe' et 'Confirmer le nouveau mot de passe' ne correspondent pas.",
```

File: src/i18n/i18n.js [hunk 10]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        share_to: "Partager à",
+        show_all_windows: "Afficher toutes les fenêtres",
+        show_hidden: 'Afficher les éléments cachés',
+        sign_in_with_puter: "Se connecter avec Puter",
+        sign_up: "S'inscrire",
+        signing_in: "Connexion…",
+        size: 'Taille',
+        sort_by: 'Trier par',
+        start: 'Démarrer',
+        taking_longer_than_usual: 'Cela prend un peu plus de temps que d'habitude. Veuillez patienter...',
```
Comment (mael-belval) [inline src/i18n/i18n.js:743]:
```suggestion
        taking_longer_than_usual: "Cela prend un peu plus de temps que d'habitude. Veuillez patienter...",
```

File: src/i18n/i18n.js [hunk 11]
```diff
@@ -614,4 +614,146 @@ window.translations = {
...
+        text_document: 'Document texte',
+        tos_fineprint: `En cliquant sur "Créer un compte gratuit", vous acceptez les <a href="https://puter.com/terms" target="_blank">conditions d'utilisation</a> et la <a href="https://puter .com/privacy" target="_blank">politique de confidentialité</a>.`,
+        trash: 'Corbeille',
+        type: 'Type',
+        undo: 'Annuler',
+        unzip: "Décompresser",
+        upload: 'Charger',
+        upload_here: 'Charger ici',
+        username: "Nom d'utilisateur",
+        username_changed: 'Nom d'utilisateur mis à jour avec succès.',
```
Comment (mael-belval) [inline src/i18n/i18n.js:753]:
```suggestion
        username_changed: "Nom d'utilisateur mis à jour avec succès.",
```

**Review by mael-belval — CHANGES_REQUESTED** [review]

**Comment by jelveh** [comment]
French has been merged by @SondreNjaastad in another PR. Thank you @Zac0511 for all the help :)

**Pull request closed without merging**